storage/s3storage/s3storage.go

```diff
@@ -58,6 +58,7 @@ func New(cfg aws.Config, bucket string, options ...Option) *S3Storage {
 	if s.Endpoint != "" {
 		s3Options = append(s3Options, func(o *s3.Options) {
 			o.BaseEndpoint = aws.String(s.Endpoint)
+			o.DisableLogOutputChecksumValidationSkipped = true
```

Comment:
This implies `DisableLogOutputChecksumValidationSkipped` is only being set if a custom S3 endpoint is enabled. Is this intented?

Otherwise, the could be set by default, regardness custom S3 endpoint or not, which may looks like this:
```go
var s3Options [func(o *s3.Options) {
	o.DisableLogOutputChecksumValidationSkipped = true
}]func(*s3.Options)
```